tests/common/TestsSetup.hs

```diff
@@ -0,0 +1,42 @@
+-----------------------------------------------------------------------------
+--
+-- Module      :  Main
+-- Copyright   :  (c) Oleg Grenrus 2015
```

Comment:
For example: https://github.com/purescript/purescript/blob/0ec9d41cfbaf5fde354ad4b3bc12bfa117d8df0b/src/Language/PureScript/Sugar/TypeClasses/Deriving.hs#L4-L5